batchflow/named_expr.py

```diff
@@ -401,6 +404,58 @@ def __repr__(self):
         return 'Unknown expression'
 
 
+class IF(NamedExpression):
+    """ Select either ``true`` or ``false``, based on ``condition``.
+    Useful for simple variables that change along the run of a pipeline.
+
+    Examples
+    --------
+    Select model mode based on the current pipeline iteration::
+        mode = IF(condition=I.current<450, true='train', false='eval')
+
+    Train the last 20% with larger batch size::
+        batch_size = IF(condition=I.ratio > 0.8, true=256, false=128)
+
+    Notes
+    -----
+    An alternative to this named expression is to use ``F``::
+        def select_batch_size(ratio):
+            return 256 if ratio > 0.8 else 128
+
+        batch_size = F(select_batch_size)(ratio=I.ratio)
```

Comment:
You can see an example of multiple lines code in the docs here: https://numpydoc.readthedocs.io/en/latest/format.html#examples